From b789a13468c102aa33c062d9270d7e2bf1705873 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 05 Aug 2018 22:23:10 +0300
Subject: [PATCH] board: habana: clean remote_load_firmware script

The intend was to use the habana_py and mxnet packages.
Now when we are not using it, but the stand alone fw flash utility,
remove the unused code.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/local_install_packages b/board/habanalabs/common/local_install_packages
index 4e44f79..e6f8c0f 100755
--- a/board/habanalabs/common/local_install_packages
+++ b/board/habanalabs/common/local_install_packages
@@ -1,9 +1,5 @@
 #!/bin/bash -l
 
-pkg_dir="$HOME/habana-pkgs"
-dl_host_base="http://jenkins:8080/view/deployment/job"
-dl_suff="lastSuccessfulBuild/artifact/*zip*"	# the '*' are not for expansion
-dl_file="archive.zip"
 pre_pkg_lst="protobuf-compiler libprotoc-dev dkms"
 habana_pkg_lst="habanalabs habanatools libhabana"
 
@@ -12,22 +8,6 @@
 	echo -e   "********************************************************"
 	echo -e   "$msg"
 	echo -e   "********************************************************"
-}
-
-function local_prepare_work_dirs() {
-	local_msg "****** Preparing directories for packages download *****"
-	mkdir -p $pkg_dir;
-	sudo umount -lrf $pkg_dir && sleep 1;
-	sudo mount -t tmpfs -o uid=$UID,dirsync,mode=0755 tmpfs $pkg_dir	|| return 1;
-}
-
-function local_dl_prep_packages() {
-	local_msg "********** Downloading and preparing packages **********"
-	cd $pkg_dir
-	wget -N ${dl_host_base}/build-habana-py/${dl_suff}/${dl_file}	|| return 1;
-	unzip -j ${dl_file}						|| return 1;
-	cd -
-	return 0;
 }
 
 function local_remove_packages() {
@@ -44,17 +24,6 @@
 	sudo apt install ${habana_pkg_lst} -y	|| return 1;
 }
 
-local_prepare_work_dirs
-if [ $? -ne 0 ]; then
-	local_msg "********* Failed preparing tmpfs for packages **********"
-	exit 1;
-fi
-
-local_dl_prep_packages
-if [ $? -ne 0 ]; then
-	local_msg "************* Failed downloading packages **************"
-	exit 2;
-fi
 
 local_remove_packages
 local_install_packages
